channels/email/accounts/managing-accounts.rst

```diff
@@ -58,20 +58,4 @@ Enabled / Disabled
 Delete
    Deleting an account removes its configuration from Zammad entirely.
 
-   .. note:: 🧹 **Additional Steps Required**
-
-      When an email account is deleted,
-      its :doc:`email aliases <secondary-addresses>` **remain in the system**. 
-      Be sure to reassign or delete them manually.
-
-      .. figure:: /images/channels/email/email-addresses-without-channel.png
-         :alt: List of orphaned email addresses
-         :scale: 60%
-         :align: center
-
-         Click on the address to assign it to another account, or click ✖ to delete it.
-
-      Groups need an assigned an address to send outgoing emails.
-      If you delete a group’s assigned address,
-      agents belonging to that group won’t be able to send emails
-      until you assign it a new one.
+   .. include:: managing-accounts.delete-note.rst
```

Comment:
Some of these commits use the `.. include::` directive to take email pages and reproduce them word-for-word in the Google pages. I did this to make sure that the pages would never fall out of sync with each other—for example, if someone updates the email page but forgets to update the corresponding Google page.

A (potentially serious) problem with this is that all the images on these pages are hardcoded to filepaths for the email docs. In other words, now the Google article on "Managing accounts" shows images for managing email accounts.

A workaround would be to store images in the same directories as the articles that use them. This way, we could use relative paths:

```
channels/email/accounts
├── managing-accounts.01-panel.png
├── managing-accounts.02-change-destination-group-for-email-account.png
├── managing-accounts.03-updating-email-account.png
├── managing-accounts.delete-note.rst
└── managing-accounts.rst

channels/google/accounts
├── managing-accounts.01-panel.png
├── managing-accounts.02-change-destination-group-for-email-account.png
├── managing-accounts.03-updating-email-account.png
├── managing-accounts.delete-note.rst
└── managing-accounts.rst

# channels/email/accounts/managing-accounts.rst
.. figure:: managing-accounts.01-panel.png
```

Reply:
Thank you so much for your effort.
Right now I'd leave the screenshot situation as be.

If we have spare time we can adjust that later on if that's okay for you. :-)